Huion_GC610/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: HGC610
```

Comment:
I failed to find anything searching for "Huion HGC610", only "Huion GC610", are you sure that's the correct model name?